config/database.yml.tara

```diff
@@ -0,0 +1,7 @@
+development:
+  adapter: postgresql
+  database: RescueRails_development
+
+test:
+  adapter: postgresql
+  database: RescueRails_test
```

Comment:
This file shouldn't get committed since it's personal.

Reply:
I'll remove the file